doc/tagbar.txt

```diff
@@ -1090,6 +1090,53 @@ containing the tag without scrolling the window.  If the tag is not visible
 in the window then the window will be scrolled and the tag (and cursor)
 placed in the location dictated by |g:tagbar_jump_offset|.
 
+                                                  *g:tagbar_highlight_method*
+g:tagbar_highlight_method~
+Default: 'nearest-stl'
```

Comment:
i wish the default is 'nearest' which kept previous default if so.

Reply:
I'm really hesitant to default to `'nearest'` as that would be the incorrect behavior for most users. In all my testing, the `.stl` flag has been set correctly even in `.go` files, so I would want to see the default behavior be what most users will expect the behavior to be. Having the option to override to `'nearest'` allows the flexibility so the user can change it if they want. It appears even in your own testing that the `.stl` flag is correct in some cases, and incorrect in others. That is likely due to some type of environmental setup difference between how the tagbar configurations are loaded or ctags output is generated. Without a way to reproduce that, I can't dig into it further, so unless many other people are seeing this behavior too, then defaulting to `'nearest-stl'` should be the expected behavior.

Reply:
i think put 'nearest' to be the default is following the general best practice which would not against previous default, and here actually not regard to 'correct' or 'incorrect', since which/where 'tag' to be highlighted actually was a personal preference..
but anyway, now we had a safeguard option, that's a bit better already. So up to you. thx.